### DISCUSSION TITLE
The "server/index.js" file?

### DISCUSSION THREAD

**Discussion by simplenotezy** [nuxt/nuxt#9958] [Questions (Nuxt 2)]
I am trying to figure out what the purpose and function of "server/index.js" file is. As described in this documentation: https://github.com/nuxt/nuxtjs.org/blob/main/content/en/deployments/azure-portal.md#running-the-webserver, it says:

 > For the latest version of Nuxt the server file is located at server/index.js

However, I went and saw the latest version on codesandbox: https://codesandbox.io/s/github/nuxt/codesandbox-nuxt/tree/master/ and there's no such file. In an older Nuxt 2 project, I also went and looked through the source code, and theres no "server/index.js".

In this current project, there is indeed a server/index.js file, but it is empty. Is this normal?

Also, I haven't been able to find more information on that file or what it does. Is it needed for static site generation?

**Comment by udithishara** [comment]
@simplenotezy  I believe this is used to spin up a server using express and use Nuxt through that as the server for SSR apps
